[2022-05-07 07:01:43,806] {processor.py:163} INFO - Started process (PID=8749) to work on /opt/airflow/dags/ej9.py
[2022-05-07 07:01:43,812] {processor.py:642} INFO - Processing file /opt/airflow/dags/ej9.py for tasks to queue
[2022-05-07 07:01:43,820] {logging_mixin.py:109} INFO - [2022-05-07 07:01:43,820] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/ej9.py
[2022-05-07 07:01:44,275] {logging_mixin.py:109} INFO - [2022-05-07 07:01:44,230] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/ej9.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ej9.py", line 3, in <module>
    from airflow.providers.apache.pig.operators.pig import PigOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-05-07 07:01:44,277] {processor.py:654} WARNING - No viable dags retrieved from /opt/airflow/dags/ej9.py
[2022-05-07 07:01:44,369] {processor.py:171} INFO - Processing /opt/airflow/dags/ej9.py took 0.578 seconds
